### PR TITLE
RYA-373 Make Benchmarks optional by profile

### PR DESCRIPTION
**Pull request by DLotts**
## Description
Made the extras/Benchmarks not part of the default build, enable with  -P benchmark 

### Tests
none

### Links
[Jira](https://issues.apache.org/jira/browse/373)

### Checklist
- [ ] Code Review
- [ ] Squash Commits

#### People To Reivew
anyone.

### CODE REVIEW

File: extras/pom.xml [hunk 1]
```diff
@@ -44,8 +44,6 @@ under the License.
         <module>rya.export</module>
         <module>rya.merger</module>
         <module>rya.giraph</module>
-       <module>rya.benchmark</module>
-       <module>rya.geoindexing</module>
```
Comment (DLotts) [inline extras/pom.xml]: Note that the geo module is already correctly removed in another PR.  I needed to test these together.

**Comment by DLotts** [comment]
asfbot build

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/489/<h2>Build result: FAILURE</span></h2>[...truncated 2.81 MB...][INFO] Apache Rya Spark Support ........................... SKIPPED[INFO] Apache Rya Web Projects ............................ SKIPPED[INFO] Apache Rya Web Implementation ...................... SKIPPED[INFO] ------------------------------------------------------------------------[INFO] BUILD FAILURE[INFO] ------------------------------------------------------------------------[INFO] Total time: 27:37 min[INFO] Finished at: 2017-09-18T17:42:44+00:00[INFO] Final Memory: 238M/2667M[INFO] ------------------------------------------------------------------------[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.4.3:shade (default) on project geo.geomesa: Error creating shaded jar: No space left on device -> [Help 1][ERROR] [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.[ERROR] Re-run Maven using the -X switch to enable full debug logging.[ERROR] [ERROR] For more information about the errors and possible solutions, please read the following articles:[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException[ERROR] [ERROR] After correcting the problems, you can resume the build with the command[ERROR]   mvn <goals> -rf :geo.geomesachannel stoppedSetting status of 9a48261373abb7c81aa788423c9374c7d51d407e to FAILURE with url https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/489/ and message: 'FAILURE 'Using context: Jenkins: clean package -Pgeoindexing

**Comment by DLotts** [comment]
asfbot build

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/492/

**Pull request closed without merging**